docs/cli/command.md

```diff
@@ -0,0 +1,18 @@
+# Command
+
+This subcommand allows you to send commands to live instances. Because of the way blocks are referenced within NIO, you must supply the target command, the name of the service, and (if you are commanding a block) the name of the block instance.
```

Comment:
not sure what you mean by subcommand—should subcommands be their own heading? is `co` the command subcommand? If so, that could be stated right here at the start, "the subcommand `co`..."